app/core/services/stt_google_v2.py

```
@@ -39,21 +39,24 @@ class STTConfig:
-        "향원정", "교태전", "강녕전", "집옥재", "자경전", "흥례문", "광화문",
-    ])
 
 
 class GoogleSTTV2:
     _LANG_MAP: dict = {
         "ko-KR": "ko", "ko-kr": "ko", "ko": "ko",
         "en-US": "en", "en-us": "en", "en-GB": "en", "en-gb": "en", "en": "en",
         "cmn-Hans-CN": "zh", "cmn-hans-cn": "zh",
+        "zh-CN": "zh", "zh-cn": "zh", "zh-TW": "zh", "zh-tw": "zh", "zh": "zh",
         "ja-JP": "ja", "ja-jp": "ja", "ja": "ja",
     }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)